core-services/release-controller/_releases/release-ocp-4.7-ci.json

```diff
@@ -58,5 +58,9 @@
       "upgradeFrom": "PreviousMinor",
       "optional":true,
       "prowJob":{"name":"periodic-ci-openshift-release-master-ci-4.7-upgrade-from-stable-4.6-e2e-aws-ovn-upgrade"}
+    },
+    "upgrade-aws-three-minors":{
```

Comment:
why did this appear here?

Reply:
As described [here][1] and in 240ee19642a91, these three-minor-hop jobs touch 4.5, and if they dropped to monthly we'd loose some signal about possible changes to 4.7.z and 4.8.z that made updates from those older minors more difficult.  Alternatively, I could keep a lower `interval` on these jobs.  Or just drop this commit and leave them at monthly.  Thoughts?

[1]: https://github.com/openshift/release/pull/24224#issuecomment-985018863

Reply:
240ee19642 -> cff3e332eb drops the three-minor-hops informers, until we straighten out a mechanism for job-ownership declaration.